src/plopp/__init__.py

```diff
@@ -14,8 +14,15 @@
 from . import widgets
 from .wrappers import plot
 
-from scipp import Variable, DataArray, Dataset
 
-setattr(Variable, 'plot', plot)
-setattr(DataArray, 'plot', plot)
-setattr(Dataset, 'plot', plot)
+def patch_scipp():
+    """
+    Running this replaces the legacy `plot` function from Scipp with the plopp `plot`
+    wrapper. This patches both the Variable, DataArray, Dataset class methods, as well
+    as the main `plot` function in the Scipp module.
+    """
+    import scipp as sc
+    setattr(sc.Variable, 'plot', plot)
```

Comment:
How does this interact with import order, or importing scipp again after calling this?

Reply:
As far as I can tell, import order doesn' seem to matter.
```Py
import scipp as sc
import plopp as pp
pp.patch_scipp()
import scipp

scipp.plot(da)
```
still gives the `plopp` plotting.